src/IdeaStatiCa.Api/Common/HttpWrapper.cs

```diff
@@ -197,7 +195,7 @@ private async Task<TResult> ExecuteClientCallAsync<TResult>(Func<HttpClient, Tas
 							// Stop the heartbeat checker
 							logger.LogTrace($"Stopping HeartbeatChecker");
 
-							if (response is { IsSuccessStatusCode: true })
+							if (response.IsSuccessStatusCode)
```

Comment:
New condition can fail on null reference exception on the response.
The first is checking if response is existing object with succsesful status code